### DISCUSSION TITLE
v0.13.0 - New Execution Providers & System.Numerics support

### DISCUSSION THREAD

**Discussion by saddam213** [saddam213/OnnxStack#75] [General]
## New Execution Providers & System.Numerics support
Added intial support for OpenVino & TensorRT Execution Providers

## What's Changed
* Add Upscale Support to UI by @saddam213 in https://github.com/saddam213/OnnxStack/pull/64
* System.Numerics.TensorPrimitives for Cpu math by @saddam213 in https://github.com/saddam213/OnnxStack/pull/72
* Add OpenVino EP support by @saddam213 in https://github.com/saddam213/OnnxStack/pull/71
* Add TensorRT ExecutionProvider by @saddam213 in https://github.com/saddam213/OnnxStack/pull/74

## Nuget Packages Updated
https://www.nuget.org/packages/OnnxStack.Core/0.13.0
https://www.nuget.org/packages/OnnxStack.StableDiffusion/0.13.0

**Full Changelog**: https://github.com/saddam213/OnnxStack/compare/v0.12.0...v0.13.0

<hr /><em>This discussion was created from the release <a href='https://github.com/saddam213/OnnxStack/releases/tag/v0.13.0'>v0.13.0 - New Execution Providers & System.Numerics support</a>.</em>

**Comment by JohnClaw** [comment]
Where is Windows UI?

**Reply by saddam213** [reply]
No real changes to the UI this week,

I did add the Upscaler UI, however there are still no ONNX upscale models so I did not see a point doing a UI release
